drop/requirementsR.txt

```
@@ -1,18 +1,15 @@
-package bioconductor version
-OUTRIDER TRUE 
-c-mertes/FRASER TRUE 1.2.0
-mumichae/tMAE TRUE 
-VariantAnnotation TRUE 
-rmarkdown FALSE 
-knitr FALSE 
-ggplot2 FALSE 
-ggthemes FALSE 
-cowplot FALSE 
-data.table FALSE  
-dplyr FALSE 
-tidyr FALSE 
-magrittr FALSE 
-devtools FALSE 
-BSgenome.Hsapiens.UCSC.hg19 TRUE 
-#MafDb.gnomAD.r2.1.hs37d5 TRUE 
-#MafDb.gnomAD.r2.1.GRCh38 TRUE 
+package	version
+gagneurlab/OUTRIDER	1.6.1
+c-mertes/FRASER	1.3.0
```

Do not use the latest devel package but rather the official stable release.

```suggestion
c-mertes/FRASER	1.2.1
```